katello/hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -37,24 +37,23 @@ def execute(commands, do_say = true, do_log = true)
       results << execute_command(command, do_say, do_log)
     end
 
-    if results.include? false
-      exit 1
-    end
+    return unless results.include? false
```

Comment:
I must say I really dislike the cop that enforces this style. Can we disable it?

Reply:
We can if you like. I :heart: guard clauses.

Reply:
They can be good and useful but not always. The cop that enforces it is too strict IMHO. I've had cases where I preferred the explicit long version because I thought it was easier to read. IMHO this is also one of them.